nibabel/tests/test_parrec.py

```diff
@@ -325,6 +340,14 @@ def test_sorting_multiple_echos_and_contrasts():
     assert_equal(np.all(sorted_types[ntotal//2:3*ntotal//4] == 2), True)
     assert_equal(np.all(sorted_types[3*ntotal//4:ntotal] == 3), True)
 
+    # check volume labels
+    vol_labels = t1_hdr.get_volume_labels()
+    assert_equal(sorted(list(vol_labels.keys())),
```

Comment:
With the ordered dict you can predict (and test for) the key order.  I think that means you can just drop the `sorted` here.